apps/e2e/lookup-storybook/src/app/country-field/country-field.component.html

```diff
@@ -0,0 +1,11 @@
+<form [formGroup]="countryForm">
+  <sky-input-box>
```

Comment:
The disabled inputs in the modern screenshots are not correct--you also need to set `[disabled]="true"` on the input box

Reply:
Which disabled input is incorrect in the screenshots? I see both of them as greyed out. I was basing this disabled functionality on [trevor's changes](https://github.com/blackbaud/skyux/pull/1460) with disabling reactive forms